compiler/damlc/tests/daml-test-files/ProposalDSL.EXPECTED.rst

```diff
@@ -10,7 +10,7 @@ Templates
 
 .. _template-proposaldsl-proposal-62786:
 
-template **Proposal**
+template (Template t) => **Proposal** t
```

Comment:
Tough question: Do we need the parentheses around `Template t`? :D

Reply:
It's an artifact of how the contexts are stored and displayed, but at least it's very consistent accross damldocs. I'll fix it at some point.